local/include/etc/confluent/docker/launch

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
is this file any different than the version in cp-kafka?

Reply:
Yes. Here we are launching both kafka and Rest-proxy. We are also checking if the kafka is up before launching the RP